librz/core/cmd/cmd_analysis.c

```diff
@@ -2546,125 +2525,43 @@ static bool convert_dot_str_to_image(RzCore *core, char *str, const char *save_p
 	return convert_dot_to_image(core, "a.dot", save_path);
 }
 
-RZ_IPI void rz_core_agraph_print_write(RzCore *core, const char *filename) {
-	convert_dotcmd_to_image(core, "aggd", filename);
-}
-
-static void cmd_agraph_node(RzCore *core, const char *input) {
-	switch (*input) {
-	case ' ': { // "agn"
-		int n_args = 0;
-		char **args = rz_str_argv(input, &n_args);
-		if (n_args < 1 || n_args > 3) {
-			rz_cons_printf("Wrong arguments\n");
-			rz_str_argv_free(args);
-			break;
-		}
-		const char *title = args[0];
-		const char *body = n_args > 1 ? args[1] : "";
-		int color = n_args > 2 ? atoi(args[2]) : -1;
-		rz_core_agraph_add_node(core, title, body, color);
-		rz_str_argv_free(args);
-		break;
-	}
-	case '-': { // "agn-"
-		char **args;
-		int n_args;
-
-		input++;
-		args = rz_str_argv(input, &n_args);
-		if (n_args != 1) {
-			rz_cons_printf("Wrong arguments\n");
-			rz_str_argv_free(args);
-			break;
-		}
-		rz_core_agraph_del_node(core, args[0]);
-		rz_str_argv_free(args);
-		break;
-	}
-	case '?': // "agn?"
-	default:
-		rz_core_cmd_help(core, help_msg_agn);
-		break;
-	}
-}
-
-static void cmd_agraph_edge(RzCore *core, const char *input) {
-	char **args;
-	int n_args;
-
-	switch (*input) {
-	case ' ': // "age"
-		args = rz_str_argv(input + 1, &n_args);
-		if (n_args != 2) {
-			rz_cons_printf("Wrong arguments\n");
-			rz_str_argv_free(args);
-			break;
-		}
-
-		rz_core_agraph_add_edge(core, args[0], args[1]);
-		rz_str_argv_free(args);
-		break;
-	case '-': // "age-"
-		args = rz_str_argv(input + 1, &n_args);
-		if (n_args != 2) {
-			rz_cons_printf("Wrong arguments\n");
-			rz_str_argv_free(args);
-			break;
-		}
-
-		rz_core_agraph_del_edge(core, args[0], args[1]);
-		rz_str_argv_free(args);
-		break;
-	case '?': // "age?"
-	default:
-		rz_core_cmd_help(core, help_msg_age);
-		break;
-	}
-}
-
-RZ_API void rz_core_agraph_print(RzCore *core, int use_utf, const char *input) {
+static void agraph_print(RzCore *core, int use_utf, RzCoreGraphFormat format) {
 	if (use_utf != -1) {
 		rz_config_set_i(core->config, "scr.utf8", use_utf);
 	}
-	switch (*input) {
-	case 0:
+	switch (format) {
+	case RZ_CORE_GRAPH_FORMAT_ASCII_ART:
 		rz_core_agraph_print_ascii(core);
 		break;
-	case 't': // "aggt" - tiny graph
+	case RZ_CORE_GRAPH_FORMAT_TINY: // "aggt" - tiny graph
 		rz_core_agraph_print_tiny(core);
 		break;
-	case 'k': // "aggk"
+	case RZ_CORE_GRAPH_FORMAT_SDB: // "aggk"
```

Comment:
I think these comments are obsolete and should be removed. Even case name provides better meaning for the purpose of this code.